hw/dv/tools/questa/sim.tcl

```diff
@@ -6,8 +6,8 @@
 # Remove leading "# " from the front of log file lines and run the test if not in gui mode.
 # This provides compatibility for log file error checking with other supported simulators within Opentitan.
 set gui 0
-if {[info exists ::env(GUI)]} {
-  set gui "$::env(GUI)"
+if {$::env(GUI)==1 || $::env(GUI_DEBUG)==1} {
+  set gui 1
```

Comment:
Is this change needed? Have you got a way to test it with a Questa licence? (I haven't!)